### PR TITLE
MM-40504 - Send ephemeral post to user on channel when `/playbooks todo` command is called

### PR DESCRIPTION
**Pull request by 97amarnathk**
#### Summary
Currently, running /playbook todo currently posts from the playbooks bot as a DM to the user.
This PR makes the response an Ephemeral Post on the channel where the command is called.

#### Ticket Link
Fixes: https://mattermost.atlassian.net/browse/MM-40504
Github issue - https://github.com/mattermost/mattermost-server/issues/19171

#### Checklist

- [ ] Telemetry updated
- [ ] ~~Gated by experimental feature flag~~
- [ ] Unit tests updated

### CODE REVIEW

File: server/app/playbook_run.go [hunk 1]
```diff
@@ -564,6 +564,10 @@ type PlaybookRunService interface {
 	// UpdateDescription updates the description of the specified playbook run.
 	UpdateDescription(playbookRunID, description string) error
 
+	// EphermalPostTodoDigestToUser gathers the list of assigned tasks, participating runs, and overdue updates,
+	// and sends an ephermal postto userID on channelID. Use force = true to post even if there are no items.
+	EphermalPostTodoDigestToUser(userID string, channelID string, force bool) error
```
Comment (gbochora) [inline server/app/playbook_run.go:569]: Just a typo. Missing space, should be 'post to'

**Review by gbochora — APPROVED** [review]
This is excellent @97amarnathk. Thanks!

**Review by Phrynobatrachus — CHANGES_REQUESTED** [review]
Hi @97amarnathk, thanks again for the PR! I fixed a couple tests that were affected by this, but it looks solid otherwise. I just have one more minor spelling nit:

`ephermal` should be `ephemeral` for all occurrences in the PR

**Comment by 97amarnathk** [comment]
@Phrynobatrachus thanks for the test fixes.
I have fixed the typo as required.

**Review by Phrynobatrachus — APPROVED** [review]

**Pull request merged**